tests/common/lib/env.bash

```diff
@@ -22,9 +22,14 @@ env.init() {
 
   ck8s init both
 
+  yq_set 'common' '.global.ck8sCloudProvider' "\"${CK8S_CLOUD_PROVIDER}\""
+
   yq_set 'common' '.global.baseDomain' '"ck8s.example.com"'
   yq_set 'common' '.global.opsDomain' '"ops.ck8s.example.com"'
 
+  yq_set 'common' '.global.scOpsDomain' '"ops.ck8s.example.com"'
+  yq_set 'common' '.global.scDomain' '"ck8s.example.com"'
+
```

Comment:
Shouln'd these just be required if you have it on another domain?

Reply:
I believe they showed up as `set-me` or empty string in the combined schema file passed to validation.

Reply:
The combined config ends up with:
```yaml
       scDomain: ""
       scOpsDomain: ""
```

Reply:
Tweaked the schema to allow empty string for now, this template did not enjoy removing the fields:

https://github.com/elastisys/compliantkubernetes-apps/blob/8f46ff29a59a2f79fff4ae9b448edcd202f57e8a/helmfile.d/values/prometheus-blackbox-exporter-wc.yaml.gotmpl#L75-L76

Reply:
Okay. I'd note that it takes a few extra lines of schema to describe that.